### PR TITLE
upgrade blackbox to v0.19.0

### LINKED ISSUE #660: Upgrade blackbox-exporter to v0.19.0
**What Chart should be upgraded?**
charts / blackbox / blackbox-exporter: [prometheus-community/prometheus-blackbox-exporter](https://github.com/prometheus-community/helm-charts/tree/main/charts/prometheus-blackbox-exporter)

v0.15.1 (2019/09/17) -> v0.19.0 (2021/05/10)

**Why should this Chart be upgraded?**

- [ ] We need to perform a security upgrade.
- [ ] We need it for a new feature: *which feature (link to blocked issue)*.
- [x] We need to keep up to prevent future issues.
- [ ] Other: *Write the reason here*

Need to upgrade this to support newer network APIs on K8s v1.22 going forward.

Also shouldn't the blackbox-exporter chart be under `upstream/`?

**Acceptance criteria**

- I checked the migration of the new Chart:
    - [ ] I upgraded a Chart and determined that no migration steps are needed.
    - [ ] I upgraded a Chart and added [migration steps](https://github.com/elastisys/compliantkubernetes-apps/blob/main/migration).
- [ ] I tested the functionality provided by the new Chart (e.g., Auth flow, Grafana dashboards, etc.)

### PR DESCRIPTION
**Pull request by Pavan-Gunda**
**What this PR does / why we need it**:
This PR upgrades Blackbox helm chart to v5.3.1 and Black-box app to v0.19.0
**Which issue this PR fixes** : fixes #660

**Public facing documentation PR** *(if applicable)*

**Special notes for reviewer**:

**Checklist:**

- [x] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [ ] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [x] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.
- Chart checklist (pick exactly one):
    - [ ] I upgraded no Chart.
    - [x] I upgraded a Chart and determined that no migration steps are needed.
    - [ ] I upgraded a Chart and added [migration steps](https://github.com/elastisys/compliantkubernetes-apps/blob/main/migration).

**Pipeline config** *(if applicable)*
If you change some config options (e.g. add/rename variable or change the default value) you may need to update the config used by the pipeline in `pipeline/config`.

### CODE REVIEW

**Comment by Pavan-Gunda** [comment]
Kube-apiserver is only giving out HTTP 200s. and hence I see kube-apiserver as down in the grafana.

**Comment by crssnd** [comment]
is the upgrade transparent? 
please check the correct box for "Is this changeset backwards compatible for existing clusters?"

**Comment by crssnd** [comment]
a general questions: why do we have a "custom" service-monitor and we do not use the one from the blackbox-exporter chart?

**Comment by Pavan-Gunda** [comment]
I don't have an answer to that. But I don't see a reason to use a custom service monitor either. Do you want me to change that?

**Comment by Xartos** [comment]
> I don't have an answer to that. But I don't see a reason to use a custom service monitor either. Do you want me to change that?

If you do that, you can also just move the chart to the `upstream` folder instead since the "elastisys chart" doesn't add anything. But if you want you can just leave this as it is and create a new issue for moving it, since that's not really the point of this issue.

**Comment by Pavan-Gunda** [comment]
I tried moving the chart to upstream and there seem to be some issues with the service monitor targets. I guess that's why we chose a custom service monitor. So, I  will merge this now and create an issue, so we can look at it in the future.

File: WIP-CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,9 @@
 - node exporter to v1.2.2 [full changelog](https://github.com/prometheus/node_exporter/blob/master/CHANGELOG.md)
 - updated metrics-server helm chart to version 0.5.2, this upgrades metrics-server image to 3.7.0 [#702](https://github.com/elastisys/compliantkubernetes-apps/pull/702)
 - Updated Dex chart to v0.6.3, and Dex itself to v2.30.0
+- Updated Blackbox chart to v5.3.1, and blackbox app to v0.19.0
+  - HTTP probe: no_follow_redirects has been renamed to follow_redirects
```
Comment (Xartos) [inline WIP-CHANGELOG.md:29]: Will you add this to `release notes` section as well since some users might use this, so I think that it should be clear.

**Review by Xartos — APPROVED** [review]
LGTM

**Review by crssnd — APPROVED** [review]

**Pull request merged**